src/cli/plugin/plugin_lister.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+module.exports = {
+  list: list
+};
+
+function list(settings, logger) {
+  fs.readdir(settings.pluginDir, function (err, files) {
```

Comment:
For cli stuff like this, it's generally best to use the `*Sync` functions so that exceptions will be thrown on error, so this becomes:

```js
const files = fs.readdirSync(settings.pluginDir);
// ...
```